sympy/physics/units/unitsystem.py

```diff
@@ -24,16 +21,15 @@ class UnitSystem(object):
     """
 
     def __init__(self, base, units=(), name="", descr=""):
+        """Build a new unit system from base and units."""
         self.name = name
         self.descr = descr
 
         # construct the associated dimension system
         self._system = DimensionSystem([u.dimension for u in base],
                                        [u.dimension for u in units])
 
-        if self.is_consistent is False:
-            raise ValueError("The system with basis '%s' is not consistent"
-                             % str(self._base_units))
+        assert self.is_consistent  # test is performed in DimensionSystem
```

Comment:
This doesn't look quite equivalent. Maybe the error message was important to display in that way.